config/configstore/accounts.go

```diff
@@ -556,62 +126,44 @@ func (acc *Account) FromJSON(data []byte) error {
 	return json.Unmarshal(data, acc)
 }
 
-// NewAccount creates a new Account instance with configs
-// TODO(ved): why do we need temp and this function ?
-func NewAccount(ethAccountName string, c config.Configuration) (config.Account, error) {
-	if ethAccountName == "" {
-		return nil, errors.New("ethAccountName not provided")
-	}
-	id, err := c.GetIdentityID()
-	if err != nil {
-		return nil, err
-	}
-	acc, err := c.GetEthereumAccount(ethAccountName)
-	if err != nil {
-		return nil, err
-	}
+type PodOperator struct {
+	URI       string           `json:"uri"`
+	AccountID *types.AccountID `json:"account_id"`
+}
 
-	cacc, err := c.GetCentChainAccount()
-	if err != nil {
-		return nil, err
+func NewPodOperator(URI string, accountID *types.AccountID) config.PodOperator {
```

Comment:
I did it this way so that `GetAccountID()` would not return an error, saves us some error handling in the long run + avoids extra computation on every call.

Reply:
Ok.